server/src/main/scala/ch.epfl.scala.index.server/Server.scala

```diff
@@ -49,12 +49,13 @@ object Server {
     // because of the sbtResolver mode
     val data = DataRepository.open()
 
-    val searchPages = new SearchPages(data, session)
+    val searchPages = new SearchPages(config.production, data, session)
```

Comment:
I was about to comment.. what is this config ? it should be config.env ^^

Reply:
^^